dev/readthedocs.md

```diff
@@ -4,28 +4,29 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
-
+   python manage.py ingest --years 2021 --terms Fall
 2. How do I then load those courses into my database?
-
+   python manage.py digest
```

Comment:
For reference, you will specify jhu like `python manage.py digest jhu` since we don't really support any other schools right now.